code/game/machinery/computer/arcade.dm

```diff
@@ -277,7 +277,6 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 	if(user.client) //mainly here to avoid a runtime when the player gets gibbed when losing the emag mode.
 		var/datum/browser/popup = new(user, "arcade", "Space Villain 2000")
 		popup.set_content(dat)
-		popup.set_title_image(user.browse_rsc_icon(icon, icon_state))
```

Comment:
these two procs were used in 20 some places, and have been commented out or empty for the past 8 years. since 12 days after browser datum was first commited.